pkg/cmd/pr/checks/output.go

```diff
@@ -49,7 +49,11 @@ func addRow(tp *tableprinter.TablePrinter, io *iostreams.IOStreams, o check) {
 		tp.AddField(o.Link)
 	} else {
 		tp.AddField(o.Name)
-		tp.AddField(o.Bucket)
+		if o.Bucket == "cancelled" {
+			tp.AddField("failed")
+		} else {
+			tp.AddField(o.Bucket)
+		}
```

Comment:
@rajhawaldar : I know this is literally what @samcoe said in the conversation, but I don't think this is literally what should go here:

```suggestion
		if o.Bucket == "cancel" {
			tp.AddField("fail")
		} else {
			tp.AddField(o.Bucket)
		}
```

Reply:
How did I miss that 🤦 Thanks @andyfeller for catching this.

Reply:
It's such a small world as I question myself the same multiple times this week 🫂